packages/nimble-components/src/patterns/tabs/template.ts

```diff
@@ -0,0 +1,67 @@
+import { html, ref, slotted, when } from '@microsoft/fast-element';
+import type { ViewTemplate } from '@microsoft/fast-element';
+import {
+    endSlotTemplate,
+    startSlotTemplate,
+    FoundationElementTemplate,
+    TabsOptions
+} from '@microsoft/fast-foundation';
+import type { Tabs } from '../../tabs';
+import type { AnchorTabs } from '../../anchor-tabs';
+import { buttonTag } from '../../button';
+import { iconArrowExpanderLeftTag } from '../../icons/arrow-expander-left';
+import { iconArrowExpanderRightTag } from '../../icons/arrow-expander-right';
+
+type TabsOrAnchorTabs = Tabs | AnchorTabs;
```

Comment:
This doesn't really lend itself to being a reusable pattern. Should we create an interface instead that Tabs and AnchorTabs are implementing?

Reply:
Added missing APIs to existing interface I created for tests.